bench/bench-local.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/sh -e
+
+scriptdir=$(CDPATH='' cd -- "$(dirname -- "$0")" && pwd -P)
+
+
```

Comment:
Extra blank line?

Reply:
I sometimes like to leave two blank lines between contants and functions, but sure, removed.